int64 request_angle 
---
sensor_msgs/Image response_image 